raml-parser-2/src/main/java/org/raml/v2/internal/impl/commons/phase/RamlFragmentGrammarTransformer.java

```diff
@@ -20,12 +20,14 @@
 import org.raml.yagi.framework.nodes.Node;
 import org.raml.yagi.framework.phase.Transformer;
 
+import static org.raml.v2.internal.utils.RamlNodeUtils.isErrorResult;
+
 public class RamlFragmentGrammarTransformer implements Transformer
 {
     @Override
     public boolean matches(Node node)
     {
-        return node instanceof RamlTypedFragment && node.getParent() != null;
+        return node instanceof RamlTypedFragment && node.getParent() != null && !isErrorResult(node);
```

Comment:
shouldn't this be added to all transformer/phases?

Reply:
It could be a good approach, but I didn't want to make a change with such impact